src/sentry/api/endpoints/project_details.py

```diff
@@ -932,6 +933,10 @@ def delete(self, request: Request, project) -> Response:
         if updated:
             scheduled = RegionScheduledDeletion.schedule(project, days=0, actor=request.user)
 
+            # Tell seer to delete all the project's grouping records
+            if features.has("projects:similarity-embeddings-delete-by-hash", project):
+                delete_project_grouping_records(project.id)
```

Comment:
Call the project record deletion endpoint

Reply:
does this spawn a celery task or call the deletion directly?

Reply:
I moved it into its own task just in case seer takes a long time